app/controllers/load_calculations_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LoadCalculationsController < ApplicationController
+  before_filter :find_testing_ground
+
+  def heat
+    render json: {
```

Comment:
FYI the JSON will [look like this](https://github.com/quintel/etmoses/issues/964#issuecomment-225800107).